afids_utils/tests/test_io.py

```diff
@@ -23,103 +23,140 @@ def valid_fcsv_file() -> PathLike[str]:
     )
 
 
-class TestGetAfid:
-    @given(afid_num=st.integers(min_value=1, max_value=32))
+@pytest.fixture
+def human_mappings() -> List[List[str] | str]:
+    with resources.open_text(
+        "afids_utils.resources", "afids_descs.json"
+    ) as json_fpath:
+        mappings = json.load(json_fpath)
+
+    return mappings["human"]
+
+
+class TestLoad:
```

Comment:
It would be nice to add a round-trip test that generates a random AfidSet, saves and loads it, then checks that the results are equal.